File: src/sdk-reference/cpp/1/index/m-delete/snippets/mDelete.cpp
std::vector<std::string> indexes;

indexes.push_back("nyc-open-data");
indexes.push_back("mtp-open-data");

try {
  std::vector<std::string> deleted = kuzzle->index->mDelete(indexes);

  std::cout << "Successfully deleted " << deleted.size() << " indexes" << std::endl;
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.getMessage() << std::endl;
}

File: src/sdk-reference/csharp/1/index/m-delete/snippets/mDelete.cs
List<string> indexes = new List<string>();

indexes.Add("nyc-open-data");
indexes.Add("mtp-open-data");

try {
  List<string> deleted = kuzzle.index.mDelete(indexes);

  Console.WriteLine("Successfully deleted " + deleted.Count + " indexes");
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.Message());
}
